quart_2/seminar_8/logger.py

```diff
@@ -0,0 +1,8 @@
+import logging
+
+logging.basicConfig(
+    level=logging.DEBUG,
+    filename='logbook.log',
+    format='[%(asctime)s] [%(levelname)s] [%(module)s] [%(funcName)s: %(lineno)d] => %(message)s',
+    datefmt='%d.%m.%Y %H:%M:%S ',
```

Comment:
Хорошая привычка, это добавлять кодировку.
```suggestion
    datefmt='%d.%m.%Y %H:%M:%S ',
    encoding='utf-8'
```